programming_examples/basic/reduce_add/Makefile

```diff
@@ -8,25 +8,27 @@
 
 include ../../makefile-common
 
+VPATH := /aie_generic
+
 ACDC_AIE = $(dir $(shell which aie-opt))/..
 
-targetname = vector_max
+targetname = add_reduce
 devicename = ipu
 col = 0
 CHESS_FLAGS=${CHESSCCWRAP2_FLAGS}
-KERNEL_LIB=${ACDC_AIE}/../../aie_kernels/aie2/
+KERNEL_LIB=../../../aie_kernels/aie2/
 
 all: build/final.xclbin build/insts.txt
 
-build/i32_add_reduce.o: ${KERNEL_LIB}/i32_add_reduce.cc
+build/reduce_add.cc.o: ${KERNEL_LIB}/reduce_add.cc
 	mkdir -p ${@D}
-	cd ${@D} && xchesscc_wrapper ${CHESS_FLAGS} -c $< -o ${@F}
+	cd ${@D} && xchesscc_wrapper ${CHESSCCWRAP2_FLAGS} -c $(<:%=../%) -o ${@F}
```

Comment:
Was CHESS_FLAGS not being set correctly? It is set to default to this AIE2 on default but when building on VCK5000 it gets set to `aie`.

Reply:
will move back to CHESS_FLAGS